java-generator/core/src/main/java/io/fabric8/java/generator/nodes/JObject.java

```diff
@@ -208,6 +208,16 @@ public GeneratorResult generateJava() {
                   new StringLiteralExpr(
                       prop.getDescription().replace("\"", "\\\""))));
         }
+
+        if (!prop.isNullable) {
```

Comment:
I miss to see an `else` branch here, is it intended?
e.g. if the field is marked as nullable we probably want to annotate it with something like: `JsonSetter( nulls = Nulls.SET)`

Have you already considered this case?

Reply:
You're right,, branch added setting `Nulls.SET`, as per the k8s specs [1] and the Jackson implementation [2].
Defaulting implementation will change this a bit once it's in, but I guess it's expected.

[1]
https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#defaulting-and-nullable

[2]
https://fasterxml.github.io/jackson-annotations/javadoc/2.10/com/fasterxml/jackson/annotation/Nulls.html